src/libs/actions/Report.js

```diff
@@ -1822,6 +1822,10 @@ function leaveRoom(reportID) {
     if (Navigation.getTopmostReportId() === reportID) {
         Navigation.goBack();
     }
+    if (report.parentReportID) {
+        Navigation.navigate(ROUTES.getReportRoute(report.parentReportID), CONST.NAVIGATION.TYPE.FORCED_UP);
```

Comment:
Coming from https://github.com/Expensify/App/issues/28087
`CONST.NAVIGATION.TYPE.FORCED_UP` has caused an edge case on web, where if you reply in a thread, **refresh the page** and select 'Leave the thread', LHN disappears from the screen